react/features/device-selection/popup.js

```diff
@@ -0,0 +1,13 @@
+import 'aui-css';
+import 'aui-experimental-css';
```

Comment:
Just checked and we can easily remove them for meet in general. However this will require some more testing and we'll definitely do this in a separate PR.